source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.md

```diff
@@ -997,6 +1093,93 @@ interface ConnectionCheckedInEvent {
    */
   connectionId: int64;
 }
+
+/**
+ *  Emitted when the connection is attempting to drain of the pending response.
+ */
+interface PendingResponseStarted {
+  /**
+   *  The ServerAddress of the Endpoint the pool is attempting to connect to.
+   */
+  address: string;
+
+  /**
+   *  The ID of the Connection.
+   */
+  connectionId: int64;
+
+  /**
+   *  The driver-generated request ID of the operation that caused the pending response state.
+   */
+  requestId: int64;
```

Comment:
Seems odd that the events emitted for an operation include a requestId that corresponds to the request that made the connection `pending response`.  What is the value in this datapoint?

If we decide to keep it - do you think a more precise name would be beneficial?  `requestId` is ambiguous - users could easily assume it refers to the request that is reading the pending response, not the operation that made the connection pending.

Reply:
The idea of reporting the original requestId is to keep tracking how long/how many draining attempt were made before success or failure. Honestly I would prefer to have BOTH, current request Id and "original timed out requestId", but it could make sense only if other check out event had "current requestId" field, which is not there =(

Reply:
Yeah, fine to keep it - thoughts on workshopping the name to make it clearer?